test/AppNotificationTests/UnpackagedTests.cpp

```diff
@@ -110,7 +110,7 @@ void UnpackagedTests::VerifyGetAllAsyncWithZeroActiveToast()
 
 void UnpackagedTests::VerifyGetAllAsyncWithOneActiveToast()
 {
-    BaseTestSuite::VerifyGetAllAsyncWithZeroActiveToast();
```

Comment:
Good catch. Would be great if there was a way to automate all that... maybe there's something taht can be done with templates in the future...